docs/section-6-contributing.md

```diff
@@ -143,3 +143,49 @@ Publishing a new release of the CLI requires these steps:
    cd cli/npm
    npm publish
    ```
+
+## Developing Individual Language Grammars
+
+Grammars for individual languages developed in the [Tree-sitter organization](https://github.com/tree-sitter/) live in `tree-sitter/tree-sitter-LANGUAGE` repositories. This section describes general practices for language grammars. Grammars developed outside of the Tree-sitter organization are strongly recommended to follow these practices for greater consistency within the Tree-sitter ecosystem.
+
+### Contributing
+
+Contributions and fixes to the grammars in the form of PRs are appreciated. To make this process pleasant for both contributors and maintainers, we ask you to follow these guidelines:
+
+- Ensure the build and all tests pass in CI, and query files are updated for the change if necessary.
+
+- Keep the PR in _draft_ mode until they are ready to be reviewed and merged. This helps maintainers to know where their attention is necessary.
+
+- Include tests that cover the proposed changes. This helps the maintainers to understand the effect of the change, and prevents regressions in the future.
+
+- If the changes concern less well known or new language features, it can be helpful to include links to documentation and/or an example program in an online playground to illustrate the change.
+
+- Avoid unnecessary changes to the structure of the syntax tree, to minimize the update work required for direct consumers of the grammar.
+
+- Check if the grammar follows a reference grammar, implementation or other documentation. If so, make sure the change does as well.
+
+### Versioning
```

Comment:
Yeah, I agree. Right now it's hard to stay on top of releasing all of the latest changes. [This PR](https://github.com/tree-sitter/tree-sitter-java/commit/81148c0902125150d616ea30190a630c86974a7a) by @Luni-4 was a good start, adding automatic publishing to crates.io.

I guess the ideal setup would be for all grammars to have something like that, and for the workflow to also publish other things as well, like the Node.js bindings and WASM builds to NPM, and at some point, the generated source code to a GitHub release, so that we could stop hosting it primarily in the git repository.